server/models/v1/show.js

```diff
@@ -2,26 +2,26 @@ const mongoose = require('mongoose');
 const Schema = mongoose.Schema;
 
 const showSchema = new Schema({
-  status: String,
+  status: String, //what are the possible values?
   show_details: {
     title: String,
     summary: String,
     description: String,
     producer: String,
     host: { type: Schema.Types.ObjectId, ref: 'User' },
     guests: [String],
-    playlist: String,
-    custom: String,
+    custom: Schema.Types.Mixed, // this will be an object that can contain any number of custom properties
   },
 
   show_start_time_utc: Date,
   show_end_time_utc: Date,
 
   master_show_uid: Number,
-  replace_show_date: Date,
+  replace_show_date: Date, //is this necessary, or redundant with show_start_time_utc?
```

Comment:
Yes, when we replace a show with a specific instance we will need to match by the master_show_uid, and the specific show to replace by date.

Reply:
@jberns got it! Does this make it redundant with `show_start_time_utc` then? Right now I'll be setting the same values for both of these.

Reply:
For the most part they would be the same.  But if on the instance you want to reschedule it for some reason, you need one of the fields to be the new date/time while still matching with the original to replace it.

Reply:
@jberns ah, that makes sense. thank you!